### PR TITLE
Data flow runs

### PR DESCRIPTION
**Pull request by rimi-itk**

### CODE REVIEW

File: assets/js/data_source.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const $ = require('jquery')
+
+$(() => {
+  const $dataSource = $('#data_source_dataSource')
+  $dataSource.change(function () {
+    // ... retrieve the corresponding form.
+    const $form = $(this).closest('form')
+    // Simulate form data, but only include the selected sport value.
```
Comment (lats1) [inline assets/js/data_source.js]: We should maybe remove the word 'sport' from this line.

File: src/Controller/DataFlowTransformsController.php [hunk 2]
```diff
@@ -67,14 +67,13 @@ public function preview(Request $request, DataFlow $dataFlow, DataTransform $tra
         $totalNumberOfSteps = $dataFlow->getTransforms()->count();
         $steps = null !== $transform ? $transform->getPosition() + 1 : 0;
 
-        $results = $this->dataFlowManager->run($dataFlow, [
+        $run = $this->dataFlowManager->run($dataFlow, [
```
Comment (lats1) [inline src/Controller/DataFlowTransformsController.php]: $run is not the best name for this variable

**Review by lats1 — CHANGES_REQUESTED** [review]

File: assets/js/data_source.js (hunk 1 above)
Comment (rimi-itk) [reply assets/js/data_source.js]: Done.

File: src/Controller/DataFlowTransformsController.php (hunk 2 above)
Comment (rimi-itk) [reply src/Controller/DataFlowTransformsController.php]: Renamed to `result`.

**Review by lats1 — APPROVED** [review]

**Pull request merged**